src/panels/lovelace/components/hui-input-list-editor.ts

```diff
@@ -0,0 +1,100 @@
+import {
+  html,
+  LitElement,
+  PropertyDeclarations,
+  TemplateResult,
+} from "lit-element";
+import "@polymer/paper-input/paper-input";
+
+import { HomeAssistant } from "../../../types";
+import { fireEvent } from "../../../common/dom/fire_event";
+
+import { EditorTarget } from "../editor/types";
+
+export class HuiInputListEditor extends LitElement {
+  protected value?: string[];
+  protected hass?: HomeAssistant;
+  protected heading?: string;
+  protected inputLabel?: string;
+
+  static get properties(): PropertyDeclarations {
```

Comment:
Please don't use this format anymore, instead use decorators. See #2711

Reply:
OK